### PR TITLE
EBNT-260 HTTP API reference for models, their artifacts and images

### PR DESCRIPTION
**Pull request by i-a-andrianov**

### CODE REVIEW

File: docs/http-api/images.rst [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+Get image
+---------
+
+Request
+^^^^^^^
+
+.. code-block::
+
+  GET /image/<:id>
```
Comment (mike0sv) [inline docs/http-api/images.rst]: '/images/...'?

File: docs/http-api/images.rst [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+Build image
+-----------
+
+Request
+^^^^^^^
+
+.. code-block::
+
+  POST /images
```
Comment (mike0sv) [inline docs/http-api/images.rst]: crud and actions on resources must be separated, i propose smt like '/images/build'

Comment (mike0sv) [reply docs/http-api/images.rst]: or hat title was misleading and it is actually not an "build action", but just a c operation from crud

File: docs/http-api/images.rst (hunk 1 above)
Comment (i-a-andrianov) [reply docs/http-api/images.rst]: fixed

File: docs/http-api/images.rst (hunk 2 above)
Comment (i-a-andrianov) [reply docs/http-api/images.rst]: How could crud and actions be separated if `Ebonite` class encapsulates them to work all together consistently? Moreover, what's the motivation to separate them in the first place?

Comment (i-a-andrianov) [reply docs/http-api/images.rst]: NB: yes, this operation is both create and build

Comment (mike0sv) [reply docs/http-api/images.rst]: ok

**Pull request merged**